apps/emqx_management/src/emqx_mgmt_api.erl

```diff
@@ -342,7 +350,11 @@ apply_total_query(QueryState = #{table := Tab}) ->
             %% return a fake total number if the query have any conditions
             0;
         Fun ->
-            Fun(Tab)
+            try
+                Fun(Tab)
+            catch
+                _Type:_Exception -> 0
```

Comment:
this may hinder exposure of other exceptions, e.g. if there is a `case_clause` somewhere.
better to move this catch down to where `ets:xxx` is called.

Reply:
Ack